### PR TITLE
Bring docs up-to-date by adding listing user's publications, listing publication's contributors and publishing into a collection

### PR DESCRIPTION
**Pull request by mikkot**
Hello @kylehg, @majelbstoat, 

Please review the following commits I made in branch 'mikko-api-publish-into-collection'.

048d0eb83f1df1faa2e5bb0221c70017f46ed620 (2015-11-11 14:31:35 -0800)
Bring docs up-to-date by adding listing user's publications, listing publication's contributors and publishing into a collection

R=@kylehg
R=@majelbstoat

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,7 +238,122 @@ Possible errors:
...
+
+Returns a full list of publications that the user is related to in some way: This includes all publications the user is subscribed to, writes to, or edits. This endpoint offers a set of data similar to what you’ll see at https://medium.com/me/publications when logged in.
+
+The REST API endpoint exposes this list of publications as a collection of resources under the user. A request to fetch a list of publications for a user looks like this:
+
+```
+GET https://api.medium.com/v1/users/{{userId}}/publications
+```
+
+The response is a list of publication objects. An empty array is returned if user doesn‘t have relations to any publications. The response array is wrapped in a data envelope.
```
Comment (kylehg) [inline README.md]: nit: the apostrophe in `doesn't` is backwards

Comment (mikkot) [inline README.md]: Fixed

**Comment by mikkot** [comment]
Bueller? Bueller?

**Comment by kylehg** [comment]
LGTM!

**Comment by mikkot** [comment]
OK merging .. @majelbstoat happy to do further changes as follow-ups if you spot anything

**Pull request merged**